File: go/1117.go
package main

import "fmt"

func main() {
  var count int
  var v, sum float64
  for {
    fmt.Scanf("%f", &v)
    if v < 0 || v > 10 {
      fmt.Println("nota invalida")
      continue
    }
    sum += v
    count += 1

    if count == 2 {
      break
    }
  }

  fmt.Printf("media = %.2f\n", sum / 2)
}

File: lua/1117.lua
local count = 0
local sum = 0.0

while true do
    local v = io.read("*n")

    if v < 0 or v > 10 then
        print("nota invalida")
    else
        sum = sum + v
        count = count + 1
    end

    if count == 2 then
        break
    end
end

print(string.format("media = %.2f", sum / 2))
